src/idsk/components/stepper/stepper.js

```diff
@@ -0,0 +1,287 @@
+
+    this.setExpanded(this.isExpanded($section), $section)
+
+    // Handle events
+    $header.addEventListener('click', this.onSectionToggle.bind(this, $section))
+
+    // See if there is any state stored in sessionStorage and set the sections to
+    // open or closed.
+    this.setInitialState($section)
+  }.bind(this))
+}
+
+Stepper.prototype.handleItemLink = function (e) {
+  var $link = e.target || e.srcElement;
+  var $currentSection = $link.closest('.idsk-stepper__section');
+  toggleClass($currentSection, 'idsk-stepper__bolder-line');
+}
+
+
+// Set individual header attributes
+Stepper.prototype.initHeaderAttributes = function ($headerWrapper, index) {
+  var $module = this.$module
+  var $span = $headerWrapper.querySelector('.' + this.$sectionButtonClass)
+  var $heading = $headerWrapper.querySelector('.' + this.$sectionHeadingClass)
+  var $summary = $headerWrapper.querySelector('.' + this.$sectionSummaryClass)
+
+  if (!$span) {
+    return;
+  }
+
+  // Copy existing span element to an actual button element, for improved accessibility.
+  var $button = document.createElement('button')
+  $button.setAttribute('type', 'button')
+  $button.setAttribute('id', this.moduleId + '-heading-' + (index + 1))
+  $button.setAttribute('aria-controls', this.moduleId + '-content-' + (index + 1))
+
+  // Copy all attributes (https://developer.mozilla.org/en-US/docs/Web/API/Element/attributes) from $span to $button
+  for (var i = 0; i < $span.attributes.length; i++) {
+    var $attr = $span.attributes.item(i)
+    $button.setAttribute($attr.nodeName, $attr.nodeValue)
+  }
+
+  $button.addEventListener('focusin', function (e) {
+    if (!$headerWrapper.classList.contains($module.$sectionHeaderFocusedClass)) {
+      $headerWrapper.className += ' ' + $module.$sectionHeaderFocusedClass
+    }
+  })
+
+  $button.addEventListener('blur', function (e) {
+    $headerWrapper.classList.remove($module.$sectionHeaderFocusedClass)
+  })
+
+  if (typeof ($summary) !== 'undefined' && $summary !== null) {
+    $button.setAttribute('aria-describedby', this.moduleId + '-summary-' + (index + 1))
+  }
+
+  // $span could contain HTML elements (see https://www.w3.org/TR/2011/WD-html5-20110525/content-models.html#phrasing-content)
+  $button.innerHTML = $span.innerHTML
+
+  $heading.removeChild($span)
+  $heading.appendChild($button)
+
+  // Add "+/-" icon
+  var $icon = document.createElement('span')
+  $icon.className = this.$iconClass
+  $icon.setAttribute('aria-hidden', 'true')
+
+  $heading.appendChild($icon)
+}
+
+// When section toggled, set and store state
+Stepper.prototype.onSectionToggle = function ($section) {
+  var $expanded = this.isExpanded($section)
+  this.setExpanded(!$expanded, $section)
+
+  // Store the state in sessionStorage when a change is triggered
+  this.storeState($section)
+}
+
+// When Open/Zatvoriť všetko toggled, set and store state
+Stepper.prototype.onOpenOrCloseAllToggle = function () {
+  var $module = this
```

Comment:
ak tu potrebuje "this" a nie "module" tak to treba nazvat $self alebo $this tak to je zmiatocne